--- efax-0.9a-001114a7/fax.orig	2007-05-06 22:55:40.000000000 +0200
+++ efax-0.9a-001114a7/fax	2007-05-06 23:04:02.000000000 +0200
@@ -881,7 +881,7 @@
               sed  -e 's/_043/#/g' -e 's/_052/*/g' \
                      -e 's/_43/#/g'   -e 's/_52/*/g' \
                      -e 's/%23/#/g'   -e 's/%2a/*/g' )
-	if ( [ "$TELNO" = "" ] || echo "$TELNO" | grep -q [^-#*0-9]) ; then
+	if ( [ "$TELNO" = "" ] || echo "$TELNO" | grep -q '[^-#*0-9]') ; then
 		echo "Illegal charactor(s) in fax number: $TELNO";ERR=2; break 2
 	fi
 
