diff --git a/efaxlib.h b/efaxlib.h
index 03e8564..8014b2b 100644
--- a/efaxlib.h
+++ b/efaxlib.h
@@ -208,7 +208,7 @@ int runor ( short *a, int na, short *b, int nb, short *c, int *pels ) ;
 /* Bit reversal lookup tables (note that the `normalbits' array
    is the one actually used for the bit reversal.  */
 
-uchar reversebits [ 256 ], normalbits [ 256 ] ;
+static uchar reversebits [ 256 ], normalbits [ 256 ] ;
 
 void initbittab(void) ;
 
